docs/en/core-services/Caching.md

```diff
@@ -180,22 +174,14 @@ alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk
 alluxio.worker.tieredstore.level0.dirs.mediumtype=MEM
 # set the quota for the ramdisk to be `100GB`
 alluxio.worker.tieredstore.level0.dirs.quota=100GB
-# set the ratio of high watermark on top layer to be 90%
-alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9
-# set the ratio of low watermark on top layer to be 70%
-alluxio.worker.tieredstore.level0.watermark.low.ratio=0.7
 # configure the second tier to be a hard disk tier
 alluxio.worker.tieredstore.level1.alias=HDD
 # configured 3 separate file paths for the second tier
 alluxio.worker.tieredstore.level1.dirs.path=/mnt/hdd1,/mnt/hdd2,/mnt/hdd3
 # defined HDD to be the medium type of the second tier
-alluxio.worker.tieredstore.level0.dirs.mediumtype=HDD,HDD,HDD
+alluxio.worker.tieredstore.level1.dirs.mediumtype=HDD,HDD,HDD
 # define the quota for each of the 3 file paths of the second tier
 alluxio.worker.tieredstore.level1.dirs.quota=2TB,5TB,500GB
-# set the ratio of high watermark on the second layer to be 90%
-alluxio.worker.tieredstore.level1.watermark.high.ratio=0.9
```

Comment:
have we totally get rid of high and low water marks? if yes, we shall mention it in this doc as many users are tuning this parameters for 2.2 and earlier

Reply:
added a note to emulation section.